adaptor-rm-webapp/pom.xml

```diff
@@ -91,6 +101,15 @@
       <scope>test</scope>
     </dependency>
 	</dependencies>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.glassfish.jersey.containers</groupId>
+				<artifactId>jersey-container-servlet-core</artifactId>
+				<version>${version.jersey}</version>
```

Comment:
what's this?

Reply:
It was an attempt to reconsile the Jersey dependency version requested by Swagger 1.5 (Jersey 2.25.1), with the latest one we want to use (2.27).
Now changing Jersey version to be 2.25.1.